app/styles/local-navbar.css

```diff
@@ -3,6 +3,7 @@
   display: flex;
   justify-content: center;
   padding: 20px;
+  background-color: white;
```

Comment:
Can we please use hex value of white ?

Reply:
changed the value to hex value #ffffff of white